main.scss

```diff
@@ -1,14 +1,183 @@
+/*
+ * NOTE:
+ * This file is used to apply our custom theme to Coral
+ * Talk V5 service.
+ *
+ * The stylesheet is built by the Origami Build Service
+ * and the URL for the CSS file is used in Coral Talk
+ * admin panel as the custom CSS URL.
+ *
+ * This is not a normal use case of Origami styles.
+ */
+
+// sass-lint:disable no-qualifying-elements nesting-depth no-important class-name-format
 @import 'src/scss/variables';
 @import 'src/scss/functions';
 @import 'src/scss/mixins';
 
+@import 'o-buttons/main';
+@import 'o-fonts/main';
+@import 'o-colors/main';
+@import 'o-typography/main';
+
 @if ($o-comments-is-silent == false) {
-	@include oComments();
+	@include oComments;
```

Comment:
We've started putting all a components css within one [primary mixin](https://github.com/Financial-Times/origami-proposals/issues/6) i.e. `oComments` with the aim of hiding silent mode from a users point of view, which can be a bit confusing. It would be great to move all this within the `oComments` mixin. E.g. like [o-forms])(https://github.com/Financial-Times/o-forms/blob/master/main.scss#L19), [o-table])(https://github.com/Financial-Times/o-table/blob/master/main.scss#L30), etc.

Reply:
This is done.